clusterctl/clusterdeployer/bootstrap/minikube/minikube.go

```diff
@@ -28,14 +28,15 @@ import (
 type Minikube struct {
 	kubeconfigpath string
 	vmDriver       string
```

Comment:
should vmDriver be removed (is it being replaced by options)?

Reply:
Agreed!